lib/Dialect/Torch/Transforms/LowerToBackendContract.cpp

```diff
@@ -247,7 +280,10 @@ class VerifyBackendContractPass
     : public VerifyBackendContractBase<VerifyBackendContractPass> {
 public:
   void runOnOperation() override {
-    if (!satisfiesBackendContract(getOperation(), /*actuallyEmitDiagnostics=*/true)) {
+    MLIRContext *context = &getContext();
```

Comment:
Is there a way we can pass in the appropriate options here? It seems a bit weird for VerifyBackendContract to say "yep" but LowerToBackendContract to disagree.

Reply:
That's a good point. Also, do you know why we have this pass at all? I don't see it used anywhere in the codebase

Reply:
@GlebKazantaev added it with the goal of ensuring that the IR produced by LTC meets the backend contract. It looks like it never got enabled -- @GlebKazantaev can you please enable it?